articles/connections/database/custom-db.md

```diff
@@ -83,47 +110,36 @@ function login(email, password, callback) {
 
-You can also [cache connections](/rules#cache-expensive-resources) such that it will survive individual execution.
-
-::: note
-If you are using [IBM's DB2](https://www.ibm.com/analytics/us/en/technology/db2/) product, [click here](/connections/database/db2-script) for a sample login script.
-:::
-
-::: note
-You can use the [auth0-custom-db-testharness library](https://www.npmjs.com/package/auth0-custom-db-testharness) to deploy, execute, and test the output of Custom DB Scripts using a Webtask sandbox environment.
-:::
-
-### Database Field Requirements
-
-Of course, your custom database will need to have fields in it that will provide the information to populate user profiles. In the above example, the script is checking for an `id`, `nickname`, `email`, and `password`.
+```
 
-For more information on Auth0 user profile schema and the fields that are expected, as well as additional ones that are available, take a look at the [Auth0 Normalized User Profile](/user-profile/normalized) documentation.
+The above script connects to a MySQL database and executes a query to retrieve the first user with `email == user.email`. With the `bcrypt.compareSync` method, it then validates that the passwords match, and if successful, returns an object containing the user profile information including `id`, `nickname`, and `email`. This script assumes that you have a `users` table containing these columns.
 
+## Add configuration parameters
 
-## 4. Add configuration parameters
+Store the credentials required to connect to your database in the **Settings** section below the script editor. Parameters stored there are available in all of your scripts.
 
-You can securely store the credentials needed to connect to your database in the **Settings** section at the bottom of the page.
+You can access parameter values using the `configuration` object in your database action scripts, for example: `configuration.PARAMETER_NAME`.
 
 ![](/media/articles/connections/database/mysql/db-connection-configurate.png)
 
-In the connection script, refer to these parameters as: `configuration.PARAMETER_NAME`. For example, you could enter:
+Heads up! You can use this section to store values which are available in all scripts. 
+The values are accessible through the global configuration object.
+
+Use the added parameters in your scripts to configure the connection. For example, in the MySQL Login template:
 
 ```js
 function login (username, password, callback) {
   var connection = mysql({
-    host     : 'localhost',
+    host     : configuration.MYSQL_HOST,
     user     : 'me',
     password : configuration.MYSQL_PASSWORD,
     database : 'mydb'
   });
 }
 ```
 
-## 5. Error handling
+## Error Handling
 
 To return an error, call the callback with an error as the first parameter:
```

Comment:
I would start this paragraph with the `There are three different errors...` and after the bullets have this sentence (`To return an error, call the callback with an error as the first parameter`) and the example (`callback(new ValidationError('email-too-long', 'Email is too long.'));`